WordPress/Classes/ViewRelated/Reader/WPStyleGuide+Reader.swift

```diff
@@ -204,55 +192,49 @@ extension WPStyleGuide {
     }
 
     public class func applyReaderCardTagButtonStyle(_ button: UIButton) {
-        let fontSize = Cards.subtextFontSize
+        WPStyleGuide.configureLabel(button.titleLabel!, forTextStyle: Cards.subtextTextStyle)
```

Comment:
This `!` scares me! This method accesses `titleLabel` three times, can we use a `guard let…` instead?